tests/backup_helper.go

```diff
@@ -2539,22 +2539,28 @@ func ValidateRestore(ctx context1.Context, restoreName string, orgID string, exp
 		// VALIDATION OF VOLUMES
 		log.InfoD("Validating Restored Volumes for the namespace (restoredAppContext) [%s] in restore [%s]", expectedRestoredAppContextNamespace, restoreName)
 
-		// Collect all volumes belonging to a context
+		// Collect all volumes belonging to a context (namespace)
 		log.Infof("getting the volumes bounded to the PVCs in the namespace (restoredAppContext) [%s] in restore [%s]", expectedRestoredAppContextNamespace, restoreName)
 		actualVolumeMap := make(map[string]*volume.Volume)
-		actualRestoredVolumes, err := Inst().S.GetVolumes(expectedRestoredAppContext)
-		if err != nil {
-			err := fmt.Errorf("error getting volumes for namespace (expectedRestoredAppContext) [%s], hence skipping volume validation. Error in Inst().S.GetVolumes: [%v]", expectedRestoredAppContextNamespace, err)
-			errors = append(errors, err)
-			continue
-		}
-		for _, restoredVol := range actualRestoredVolumes {
-			actualVolumeMap[restoredVol.ID] = restoredVol
+		for _, appContext := range expectedRestoredAppContexts {
+			log.Infof("appContext.App.NameSpace [%s] expectedRestoredAppContextNamespace [%s] == [%v]", appContext.App.NameSpace, expectedRestoredAppContextNamespace, appContext.App.NameSpace == expectedRestoredAppContextNamespace)
```

Comment:
Might not be necessary

Reply:
Sorry @kphalgun-px , do you mean the log message present in line #2546?

Reply:
Yes.

Reply:
Addressed